.pipelines/networkobservability/pipeline.yaml

```diff
@@ -0,0 +1,145 @@
+pr: none
+trigger: none
+
+schedules:
+- cron: '0 0 * * *'
+  displayName: Daily midnight Cilium with Hubble
+  branches:
+    include:
+    - master
+
+variables:
+  clustername: ciliumhubble-$(Build.SourceBranchName)-$(Build.BuildId)
+  cilium_version: v1.14.4
```

Comment:
nit: Are we sure this should be hardcoded?

Reply:
This is bound to the folder structure. It doesn't make sense to be dynamic (yet) until we come up with a proper strategy.